test/e2e/testnet/testnet.go

```diff
@@ -354,58 +370,30 @@ func (t *Testnet) Start() error {
 			if i == 9 {
 				return fmt.Errorf("failed to start node %s", node.Name)
 			}
-			time.Sleep(time.Second)
+			fmt.Printf("node %s is not synced yet, waiting...\n", node.Name)
+			time.Sleep(100 * time.Millisecond)
 		}
 	}
 	return nil
 }
 
 func (t *Testnet) Cleanup() {
-	for _, node := range t.nodes {
-		if node.Instance.IsInState(knuu.Started) {
-			if err := node.Instance.Stop(); err != nil {
-				log.Err(err).
-					Str("name", node.Name).
-					Msg("node  failed to stop")
-				continue
-			}
-			if err := node.Instance.WaitInstanceIsStopped(); err != nil {
-				log.Err(err).
-					Str("name", node.Name).
-					Msg("node  failed to stop")
-				continue
-			}
-		}
-		if node.Instance.IsInState(knuu.Started, knuu.Stopped) {
-			err := node.Instance.Destroy()
-			if err != nil {
-				log.Err(err).
-					Str("name", node.Name).
-					Msg("node  failed to cleanup")
-			}
```

Comment:
There is no need to stop the instance before destroying, we can simply destroy it.